### DISCUSSION TITLE
Auto Matching for built-in server

### DISCUSSION THREAD

**Discussion by glawleschkoff** [LeoKlaus/Paperparrot#200] [Feature Requests]
It would be great to have the the auto match algorithm from Paperless ngx available for tag, document type and correspondent **for the built-in server**

**Comment by LeoKlaus** [comment]
Hey Glawleschkoff,

thank you for reaching out!
Automatic matching is already selectable for attributes if you're connected to a Paperless-server. If you can't see the option, that's a bug.
![image](https://github.com/LeoKlaus/Paperparrot/assets/66492595/dd96a131-454a-4d79-91f3-5c3b5edff97e)
